ovos_dinkum_listener/service.py

```diff
@@ -681,7 +682,36 @@ def _save_stt(self, audio_bytes, stt_meta, save_path=None):
             stt_audio_dir = Path(f"{self.default_save_path}/utterances")
         stt_audio_dir.mkdir(parents=True, exist_ok=True)
 
-        filename = hash_sentence(stt_meta["transcription"])
+        listener = self.config.get("listener")
+
+        # What is a good default?
+        # Where does the documentation for this feature go?
+        # TODO: document the listener.filename_template option
```

Comment:
docs go here https://github.com/OpenVoiceOS/ovos-technical-manual

i think the current default is good

Reply:
I'm thinking this might not need to go in the technical manual. Nothing seems to reference the listener config in it already. The docs in the config file itself will probably work fine for now. I made a PR here: https://github.com/OpenVoiceOS/ovos-config/pull/157